sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2161,6 +2161,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val LEGACY_MSSQLSERVER_NUMERIC_MAPPING_ENABLED =
+    buildConf("spark.sql.legacy.mssqlserver.numericMapping.enabled")
```

Comment:
For such a change, could we have an item in the migration guide?

Reply:
Sure. Can we handle that after releasing 2.4.5 since It's a documentation change.

Reply:
Thank you!